.github/workflows/ci.yml

```diff
@@ -22,7 +22,11 @@ jobs:
       # and hyphens.
       run: echo "TAG=branch-${GITHUB_REF_NAME//[^A-Za-z0-9._-]/-}" | tee -a $GITHUB_ENV
 
-    - run: ./devel/build $TAG
+    - uses: docker/setup-qemu-action@v2
+
+    - run: ./devel/start-localhost-registry
+
+    - run: ./devel/build-to-localhost $TAG linux/amd64,linux/arm64
 
```

Comment:
With the release of [Augur 18.0.0](https://github.com/nextstrain/augur/releases/tag/18.0.0) which makes cvxopt a required dependency, another dependency of that [fails to compile](https://github.com/nextstrain/docker-base/actions/runs/3139336657/jobs/5099809084):

```
gcc -pthread -shared -Wl,--strip-all build/temp.linux-aarch64-3.7/src/C/base.o build/temp.linux-aarch64-3.7/src/C/dense.o build/temp.linux-aarch64-3.7/src/C/sparse.o -L/usr/lib -L/usr/local/lib -lm -llapack -lblas -lpython3.7m -o build/lib.linux-aarch64-3.7/cvxopt/base.cpython-37m-aarch64-linux-gnu.so
/usr/bin/ld: cannot find -llapack
/usr/bin/ld: cannot find -lblas
collect2: error: ld returned 1 exit status
error: command 'gcc' failed with exit status 1
```

This is similar to https://github.com/nextstrain/docker-base/pull/50#discussion_r956218475, but this time we can't ignore the dependency. Options:

1. Install whatever is necessary to compile properly (maybe `sudo apt-get install liblapack-dev libblas-dev`?)
2. Install pre-built `linux-aarch64` binaries from [conda-forge](https://anaconda.org/conda-forge/cvxopt).

I'm planning on (1) since (2) seems like a bigger change to the Dockerfile.

Reply:
(1) or its equivalent seems best, yes. There are several Ubuntu packages providing libblas.so and liblapack.so, which should roughly match what Debian in the container image here has too.

![image](https://user-images.githubusercontent.com/79913/192846757-579ac86e-cbd1-40b2-8130-ed6a3983959f.png)

In prior versions of this image (back on Alpine, so packaging names are a bit different), we compiled cvxopt like so:

https://github.com/nextstrain/docker-base/blob/3917c1b9aa0aa19e694f94f6bd3e56418f248269/Dockerfile#L89-L94

This used openblas (Ubuntu package `libopenblas-dev` above) to provide libblas.so and liblapack.so and also installed libsuitesparse for use by cvxopt (which itself relies on blas and lapack).

Reply:
It's too bad that cvxopt isn't [using cibuildwheels for multi-arch wheels](https://cibuildwheel.readthedocs.io/en/stable/options/#archs).

Reply:
Interesting historical info!

2bc2fa1bf13f12174a667982f40ff63d710fad86 should fix this.